sdk/appconfiguration/app-configuration/test/public/secretReference.spec.ts

```diff
@@ -173,7 +173,9 @@ describe("AppConfigurationClient - SecretReference", () => {
         0,
         "Unexpected number of SecretReferences seen after updating"
       );
-      await client.setReadOnly({ key: secondSetting.key, label: secondSetting.label }, false);
+      if (secondSetting.isReadOnly) {
```

Comment:
No, Look at line 166, secondSetting is not updated, instead the value is readOnly is.

Can you check this better?